### ISSUE TITLE
Support CDK Workloads (typescript)

### ISSUE THREAD

**Issue by OlafConijn** [OlafConijn/AwsOrganizationFormation#55]
Hi, in order to add CDK support i was thinking of the following task type:

```
ServerlessWorkload:
  Type: update-cdk                                   # type
  Path: ./workload/                                  # used to package / calculate hash
  RunNpmInstall: true                                # see #1
  RunNpmBuild: true                                  # see #2 
  OrganizationBinding:
    AccountsWithTag: selective-few
  MaxConcurrentStacks: 10
  FailedStackTolerance: 10
```

Questions:
1. `RunNpmInstall` will run npm ci or i (depdending on whether there is a package.lock). For other languages we can make a RunPipInstall, etc.
2. In the examples (https://github.com/aws-samples/aws-cdk-examples) i see that a build command needs to be ran. does it make sense to pass the npm action as a parameter? or commit to a convention.

**Comment by eduardomourar** [comment]
It would be nice to be able to pass additional arguments (such as `--verbose`, `STACK_NAME`, etc) for the `cdk deploy` command.

**Comment by OlafConijn** [comment]
@eduardomourar, added `AdditionalCdkArguments` to the task configuration in 0.9.4-beta.14

**Comment by eduardomourar** [comment]
AWS has a proposal to improve the CI/CD process for the CDK: https://github.com/aws/aws-cdk-rfcs/blob/master/text/0049-continuous-delivery.md

**Comment by OlafConijn** [comment]
implemented in version 0.9.4
